tests/utils.py

```diff
@@ -130,3 +172,27 @@ def assert_graph_structures(graph_structure: Dict, snapshot_path: str) -> None:
         assert (
             relationship == graph_relations[snapshot_relations.index(relationship)]
         ), f"Properties of relationships on the same index of graph and snapshot is not equal\n\n{get_dicts_differences(relationship, graph_relations[snapshot_relations.index(relationship)])}\nIn snapshot:\n{relationship}\nIn graph:\n{graph_relations[snapshot_relations.index(relationship)]}"
+
+
+def assert_action_inputs(ca: CompositeAction, ca_d: Dict):
```

Comment:
Maybe we need to split the utils to unittests utils and integration utils

Reply:
`get_sorted_lists_of_nodes_and_relationships` is used by the unit test function and integration test function. I think you are right but lets save this for another PR. This one is already of topic to test logic.